installCatalog.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# use the command line interface to install standard actions deployed
+# automatically
+#
+# To run this command
+# ./installCatalog.sh  <AUTH> <APIHOST> <KAFKA_TRIGGER_HOST> <KAFKA_TRIGGER_PORT>
+# AUTH and APIHOST are found in $HOME/.wskprops
+
+set -e
+set -x
+
+: ${OPENWHISK_HOME:?"OPENWHISK_HOME must be set and non-empty"}
+WSK_CLI="$OPENWHISK_HOME/bin/wsk"
+
+if [ $# -eq 0 ]
+then
+echo "Usage: ./installCatalog.sh <authkey> <apihost> <kafkatriggerhost> <kafkatriggerport>"
```

Comment:
@dubeejw - please feel free to make any changes to this file you see fit.  I just threw an install script together real quick for testing purposes.  I knew you would be making updates to it (annotations, package and actions names, etc.....)